scripts/release_info.py

```diff
@@ -243,14 +245,47 @@ def main():
             for item in storage_changes:
                 out.write('* %s  \n' % item)
 
-        out.write('\n### Authors:\n')
+        past_authors = {
+            log.email: log.author for log in past_logs
+        }
+        release_authors = {(log.author, log.email) for log in new_release_logs}
+
+        new_authors = sorted(set(
+            [(name, email) for name, email in release_authors
+             if email not in past_authors]))
+        existing_authors = sorted(set(
+            [(name, email) for name, email in release_authors
+             if email in past_authors]))
+        new_author_names = [name for name, _ in new_authors]
+        existing_author_names = [name for name, _ in existing_authors]
+
         # TODO: duplicate author handling due to email changes
```

Comment:
Hm, I'm not sure what this means but maybe it's similar to what I just commented.

Reply:
Not sure either; it's leftover from before.